docs/command_line_tutorial.rst

```diff
@@ -0,0 +1,273 @@
+Buildtest Command Line Tutorial
+=================================
+
+For this session, we assume you have :ref:`installed buildtest <installing_buildtest>` on your system.
+You can check if ``buildtest`` command is available by running::
+
+    $ buildtest --help
+
+If you receive an error please go back and re-install buildtest.
+
+If you are new to buildtest you can review the :ref:`quick start guide <quick_start>` to learn the basics
+of buildtest.
+
+
+Building Test
+---------------
+
+The ``buildtest build`` command is used for running a test on your system given a :ref:`buildspec <what_is_buildspec>`
+file (YAML). The most common way to build a test is specifying a file path via ``buildtest build -b <path>``. To get started,
+let's build our first test by running the following::
+
+    buildtest build -b $BUILDTEST_ROOT/tutorials/hello_world.yml
+
+If you ran this successfully, you should see the following output
```

Comment:
Lines 22-24 can be removed as the command is already shown and executed in dropdown below.

Reply:
fixed